### PR TITLE
LPD-4174 functional test

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:playwright

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java [hunk 1]
```diff
@@ -126,8 +126,8 @@ private Map<String, Object> _getTermValues(
 				SimpleDateFormat simpleDateFormat = new SimpleDateFormat(
 					"yyyy-MM-dd");
 
-				Timestamp timestamp = (Timestamp)termValues.get(
-					objectField.getName());
+				Timestamp timestamp = new Timestamp(
+					(Long)termValues.get(objectField.getName()));
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java:130]: Please, remember to consider this review [here](https://github.com/liferay-commerce/liferay-portal/pull/4838/files/08a59747ba023b850f13fee1fdf385cb5f18538c..506d15196c922900e451fb427f5cd9fef51d2b78#r1553912802). Thank you :smile:

**Comment by dsitu** [comment]
Opened up a new PR with changes [here](https://github.com/liferay-commerce/liferay-portal/pull/4842).

**Pull request closed without merging**